src/core/mac/sub_mac.cpp

```diff
@@ -261,14 +264,22 @@ void SubMac::HandleReceiveDone(RxFrame *aFrame, Error aError)
         UpdateFrameCounter(aFrame->mInfo.mRxInfo.mAckFrameCounter);
     }
 
-#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE
+#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE || OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
```

Comment:
```suggestion
#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
```
Note that `OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE` can only be enabled when `OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE`. We have a compile-time check for ensuring this. So we can just use `OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE` here.